### ISSUE TITLE
Always use realtime pulse loading for AWG8 lutmans 

### ISSUE THREAD

**Issue by AdriaanRol** [DiCarloLab-Delft/PycQED_py3#482]
Problem description
    - "conventional" loading is very slow (~2s per AWG channel -> ~16s per experiment) 
    - conventional and realtime waveforms are not identical (LSB error qusurf issue #57) -> very relevant for CZ tuneup 
    - Currently loads of hacks to use realtime loading only with a single waveform (waveform 0) -> introduces inconsistencies in CZ tuneup and uncertainty as to what calibration is valid. 

Approach to address this problem 
    - Recently discovered that realtime loading works for "arbitrary" waveforms as long as the DIO program is loaded beforehand 
    - Always use realtime loading as the default. 
    
I will start by implementing this in the MW lutman. Additionally I will store the hash of the uploaded program so that I can verify if the program is loaded and auto load if it is not loaded. 

In summary (for the mw_lutman) the flow will be something like this 
``` python 
def upload_pulses()
    gen pulses () 
    if AWG.program_hash != expected_hash : 
              upload program 

    load_pulses realtime 
```

**Comment by AdriaanRol** [comment]
Subtasks 
- [ ] Add program hash parameters to AWG8 driver 
- [ ] Update loading commands in mw_lutman
- [ ] Update loading commands in flux_lutman

**Comment by AdriaanRol** [comment]
This was implemented quite some time ago.
